services/agent/src/simcore_service_agent/_settings.py

```diff
@@ -0,0 +1,43 @@
+from typing import Final
+
+from models_library.basic_types import LogLevel
+from pydantic import Field, NonNegativeInt, validator
+from settings_library.base import BaseCustomSettings
+from settings_library.r_clone import S3Provider
+
+_MIN: Final[NonNegativeInt] = 60
+
+
+class ApplicationSettings(BaseCustomSettings):
+    LOGLEVEL: LogLevel = Field(LogLevel.INFO)
```

Comment:
please set this up **analogously** to the rest of apps: see e.g. 

https://github.com/ITISFoundation/osparc-simcore/blob/7be3511dbe227a8c484d41cdd460f5b0e222d96b/services/web/server/src/simcore_service_webserver/application_settings.py#L85